tests/test_processors.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from lxml.html import fromstring
+from parsel import Selector
+from web_poet import field
+from zyte_parsers import Breadcrumb as zp_Breadcrumb
+
+from zyte_common_items import BasePage, Breadcrumb
+from zyte_common_items.processors import breadcrumbs_processor
+
+breadcrumbs_html = """
+<div class="pagesbar">
+  <ul>
+    <li><a href="/blog/">Home</a></li>
+    <li><a href="/blog/about/">About</a></li>
+  </ul>
+</div>
+"""
+
+breadcrumbs_expected = [
+    Breadcrumb(name="Home", url="http://www.example.com/blog/"),
+    Breadcrumb(name="About", url="http://www.example.com/blog/about/"),
+]
+
+
+@pytest.mark.parametrize(
+    "input_value",
+    (
+        breadcrumbs_expected,
+        fromstring(breadcrumbs_html),
+        Selector(text=breadcrumbs_html),
```

Comment:
I think it'd be nice to have a test for the "happy path" - a page object like this:

```py
class MyProductPage(ProductPage):
    @field
    def breadcrumbs(self):
        return self.css(".breadcrumbs")
```

If I'm not mistaken, it'd actually fail :) SelectorList vs Selector

Reply:
Another scenario to test is returning a value which is different from the value which processor would extract, to ensure that the processor doesn't run.